packages/flutter/lib/src/material/material_localizations.dart

```diff
@@ -4,18 +4,21 @@
 
 import 'package:flutter/widgets.dart';
 
+import 'typography.dart';
+
 /// Interface for localized resource values for the material widgets.
 ///
 /// This class provides a default placeholder implementation that returns
 /// hard-coded American English values.
 class MaterialLocalizations {
   /// Create a placeholder object for the localized resources of material widgets
   /// which only provides American English strings.
-  const MaterialLocalizations();
+  const MaterialLocalizations({this.locale: const Locale('en', 'US')})
```

Comment:
nit: space (or newline, typically preferred) after `{` for named arguments (and matching whitespace on the `}` side)